modules/ROOT/pages/configure-custom-serializers.adoc

```diff
@@ -0,0 +1,171 @@
+= Configure Custom Serializers
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can change the serialization mechanism to improve functionality and performance when Mule runtime engine executes any of the following processes:
+
+* Read from or write to a persistent ObjectStore
+* Read from or write to a persistent VM or JMS queue
+* An object distributes through a Mule cluster
+* Read from or write to an object from a file
+
+By default, Mule uses plain old Java serialization. You can configure `defaultObjectSerializer` in your Mule application to specify a different serialization mechanism such as the Kryo serializer or any other custom serializer.
+
+The Batch module always uses Kryo serialization, so the module's behavior is not affected by the configured serializer.
+
+== Compatibility
+
+If you are running Mule runtime engine (Enterprise Edition), you can configure the Kryo serializer to improve performance.
+
+If you are using Mule Kernel (Community Edition), you can create a custom serializer using the Serialization API.
+
+== Configure the Kryo Serializer
+
+Mule provides an implementation of `ObjectSerializer` which relies on the Kryo framework. Using Kryo provides the following benefits:
+
+* Better performance +
+Kryo is much faster than Java serialization
```

Comment:
Add . (dot) at the end of sentence. _Kryo is much faster than Java serialization._